repository/build.gradle

```diff
@@ -1,25 +1,29 @@
 jar {
-    baseName = 'definition-designer-repository'
+    enabled = true
+    manifest {
+        attributes 'Implementation-Title': project.name,
+                'Implementation-Version': project.version
+    }
+    archiveName = 'definition-designer-repository'
 }
 
-bootRepackage {
+bootJar {
     enabled = false
 }
 
 dependencies {
     compile("org.springframework.boot:spring-boot-starter-data-jpa") {
         exclude group: 'org.apache.tomcat', module: 'tomcat-jdbc'
     }
-    compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '0.6.0'
-    compile "commons-beanutils:commons-beanutils"
+    compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '1.0.4'
+    compile "commons-beanutils:commons-beanutils:1.9.3"
     compile "org.apache.commons:commons-lang3"
     compile "io.springfox:springfox-swagger2:${springfoxSwaggerVersion}"
     compile "io.springfox:springfox-swagger-ui:${springfoxSwaggerVersion}"
     testCompile "org.springframework.boot:spring-boot-test"
     compile "javax.validation:validation-api:2.0.1.Final"
     compile "org.hibernate.validator:hibernate-validator:${hibernateValidatorVersion}"
     compile "org.glassfish:javax.el:3.0.1-b11"
-    testCompile "org.liquibase:liquibase-core"
     compile group: 'com.vladmihalcea', name: 'hibernate-types-52', version: '2.4.1'
```

Comment:
I thought there were only a few places that need liquibase.  By declaring compile dependency seemed to be a little too much

Reply:
Agreed, reverting to testCompile and removing compile dependency from the parent project